zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -747,6 +754,18 @@ private static boolean publishDatasets(Collection collection) throws IOException
         });
     }
 
+    private static void sendToKafka(Collection collection) throws IOException {
+        List<String> uris = collection.getReviewed().uris();
+        try {
+            kafkaServiceSupplier.getService().produceContentPublished(collection.getId(), uris);
+        } catch (Exception e) {
+            error().data("collectionId", collection.getDescription().getId()).data("publishing", true)
+                    .logException(e, "failed to send content-published kafka events");
+            PostMessageField msg = new PostMessageField("Error", e.getMessage(), false);
+            collectionAlarm(collection, "Failed to send content-published kafka events", msg);
```

Comment:
Can you pull the latest develop if you haven't already - Sandy has updated how we send slack notifications and I think this class (SlackNotification) is now deprecated. We've updated this class already so it should have everything required to use the new approach. There should be examples already but as a rough guide it looks like:
```
 String channel = Configuration.getDefaultSlackAlarmChannel();
Notifier notifier = zebedee.getSlackNotifier();
notifier.sendCollectionAlarm(collection, channel, "you message here, ex);
```